src/components/Arrow/component.js

```diff
@@ -1,64 +1,95 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Fragment } from 'react';
 import { getComponentTestId } from 'utils/test-utils';
 import { noop } from 'lodash';
 import { omit } from 'ramda';
-import { positionPropTypes } from 'models';
+import { positionPropTypes, contextMenuItemPropTypes } from 'models';
+import ContextMenuTrigger from 'components/ContextMenuTrigger';
+import { CONTEXTMENU_TYPES } from 'constants/contextmenu';
+import ContextMenuItems from 'components/ContextMenuItems';
 
 const style = {
   transition: 'stroke-opacity 0.2s',
 };
 const stylePath = { cursor: 'pointer' };
-const omitProps = omit(['markEnd', 'markEndStyle']);
+const omitProps = omit([
```

Comment:
Instead of an omit isn't better to have a `pick` explicitly telling what keys you need? (Unless it's a long list)